spec/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_images_spec.rb

```diff
@@ -0,0 +1,146 @@
+require Rails.root.join('db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations' \
+                        '/Methods.class/__methods__/available_images.rb').to_s
+
+describe ManageIQ::Automate::Cloud::Orchestration::Operations::AvailableImages do
+  let(:default_desc) { "<select>" }
+  let(:default_desc_blank) { "<none>" }
+  let(:ems) do
+    @hw1 = FactoryGirl.create(:hardware, :guest_os => 'windows')
+    @hw2 = FactoryGirl.create(:hardware, :guest_os => 'linux')
+    @img1 = FactoryGirl.create(:template_openstack, :uid_ems => 'uid1', :hardware => @hw1)
+    @img2 = FactoryGirl.create(:template_openstack, :uid_ems => 'uid2', :hardware => @hw2)
+    FactoryGirl.create(:ems_openstack, :miq_templates => [@img1, @img2])
+  end
+  let(:root_hash) do
+    { 'service_template' => MiqAeMethodService::MiqAeServiceServiceTemplate.find(service_template.id) }
+  end
+  let(:root_object) { Spec::Support::MiqAeMockObject.new(root_hash) }
+  let(:ae_service) do
+    Spec::Support::MiqAeMockService.new(root_object).tap do |service|
+      current_object = Spec::Support::MiqAeMockObject.new
+      current_object.parent = root_object
+      service.object = current_object
+    end
+  end
+
+  context "workspace has no service template" do
+    let(:root_hash) { {} }
+
+    it "provides only default value to the image list" do
+      described_class.new(ae_service).main
+
+      expect(ae_service["values"]).to eq(nil => default_desc_blank)
```

Comment:
@pkomanek 
Since this is a new spec file we could also reduce the checks by using a shared example
https://github.com/ManageIQ/manageiq/blob/master/spec/lib/miq_automation_engine/models/miq_ae_git_import_spec.rb#L68
And then calling it within a context as
https://github.com/ManageIQ/manageiq/blob/master/spec/lib/miq_automation_engine/models/miq_ae_git_import_spec.rb#L79

what ever needs to change for the individual test you define in the context and then call it_behvaes_like